src/main/java/com/avides/springboot/springtainer/common/cleanup/EmbeddedContainerCleanupAutoConfiguration.java

```diff
@@ -56,17 +56,17 @@ private int removeStaleContainers(ContainerCleanupProperties properties)
             {
                 for (Container container : dockerClient.listContainersCmd().exec())
                 {
-                    if (container.getLabels().containsKey(TESTCONTAINER_STARTED))
+                    if (container.getLabels().containsKey(SPRINGTAINER_STARTED))
```

Comment:
**Warning**
If we only use the new labels, old containers (Spring Boot test containers) will not be cleaned up. Therefore, there is a higher probability that some old containers are still available!
If everyone accepts that, we can ignore it.

Reply:
I accept to ignore it.

Reply:
If you update one container, I assume that you don't have the old one running at this task. So, I seem we can ignore that.